deploy/helm/sumologic/templates/statefulset.yaml

```diff
@@ -146,9 +146,9 @@ spec:
         - name: K8S_METADATA_FILTER_VERIFY_SSL
           value: {{ .Values.sumologic.k8sMetadataFilter.verifySsl | quote }}
         - name: K8S_METADATA_FILTER_BEARER_CACHE_SIZE
```

Comment:
can we remove "BEARER" from these env variable names as well?

Reply:
(for the non-Helm users' benefit)

Reply:
ok. wondering why was it added in the first place as the param name is `cache_size` for the metadata filter plugin as well.

Reply:
yeah, not sure